packages/manager/modules/iam/src/components/identities/userIdentities/userIdentities.template.html

```diff
@@ -1,10 +1,55 @@
 <section>
+    <iam-select-modal
+        data-current-identities="$ctrl.identities"
+        data-ng-if="$ctrl.isSelectingUsers"
+        data-close="$ctrl.closeSelectUsersModal()"
+        data-get-options="$ctrl.getUserList()"
+        data-on-add-options="$ctrl.onAddUsers(selectedOptions)"
+        data-heading="{{:: 'iam_identities_users_modal_title' | translate}}"
+        data-empty-state-message="{{:: 'iam_identities_users_modal_empty_state' | translate}}"
+        data-search-placeholder="{{:: 'iam_identities_users_modal_search_placeholder' | translate}}"
+        data-search-filter="$ctrl.constructor.userSearchFilter(option, query)"
+    >
+        <iam-checkbox-value>
+            <span data-ng-bind="$parent.option.login"></span>
+        </iam-checkbox-value>
+        <iam-checkbox-description>
+            <div>
+                <span data-translate="iam_identities_users_email"></span>
+                <b data-ng-bind="$parent.option.email"></b>
+            </div>
+            <div>
+                <span data-translate="iam_identities_users_group"></span>
+                <b data-ng-bind="$parent.option.group"></b>
+            </div>
+            <div>
+                <span data-translate="iam_identities_users_status"></span>
+                <b
+                    data-ng-bind="'iam_identities_users_status_' + $parent.option.status | translate"
```

Comment:
```suggestion
                    data-translate="{{ 'iam_identities_users_status_' + $parent.option.status}}"
```

Reply:
this fix does not work with a transcluded component, `$parent.option.status` is undefined.